src/backend/distributed/sql/udfs/citus_database_size/latest.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_database_size(
```

Comment:
Minor: `citus_total_database_size`

Reply:
What would be the difference between database and total database?

Reply:
We also have `citus_node_database_size` so when there is `citus_node_database_size`  `citus_database_size`  could mean either the total or local. I guess if there is only `citus_database_size` then it is clear but having the `citus_node_database_size` adds the ambiguity